server/src/main/java/org/jetbrains/bsp/bazel/server/sync/ProjectProvider.java

```diff
@@ -0,0 +1,53 @@
+package org.jetbrains.bsp.bazel.server.sync;
+
+import java.util.ArrayList;
+import org.jetbrains.bsp.bazel.server.sync.model.Project;
+
+public class ProjectProvider {
+  private Project project;
+  private final ProjectResolver projectResolver;
+  private final java.util.List<ProjectChangeListener> listeners = new ArrayList<>();
```

Comment:
same as above

Reply:
generally this is because I prefer vavr List imported. I see that these files do not have conflicts, It must be result of some refactoring like moving methods around or removing some code. Will clean up once the PR is fully reviewed.